requirements.txt

```diff
@@ -17,4 +17,5 @@ sklearn
 tables
 h5py
 helics
-pytest
\ No newline at end of file
+pytest
```

Comment:
I downloaded this branch and tried to run it.  I got some import errors related to Swagger and maybe some others.  You'll need to make some updates here.